elements/pl-matrix-component-input/pl-matrix-component-input.mustache

```diff
@@ -44,10 +44,14 @@
         });
     });
 </script>
-<span id="pl-matrix-component-input-submission-{{uuid}}">
+<span id="pl-matrix-component-input-submission-{{uuid}}" class="pl-matrix-component-input-invalid">
     {{#label}}<span>{{label}}</span>{{/label}}
-    <a role="button" class="btn btn-light btn-sm" data-placement="auto" data-trigger="focus" data-toggle="popover" data-html="true" title="Error" tabindex="0" data-content="{{parse_error}}{{#raw_submitted_answer}}<hr><h5><small class='text-uppercase'>What you submitted</small></h5><pre>{{raw_submitted_answer}}</pre>{{/raw_submitted_answer}}">INVALID
-      <span><i class="fa fa-question-circle" aria-hidden="true"></i></span>
+
+    {{#raw_submitted_answer}}{{{raw_submitted_answer}}}{{/raw_submitted_answer}}
+    
+    <span class="badge text-danger badge-invalid"><i class="fa fa-exclamation-triangle" aria-hidden="true"></i> Invalid</span>
```

Comment:
I love the new styling! It's clearer and also looks much more elegant. Double win!